ras/supportconfig.py

```diff
@@ -11,8 +11,9 @@
 #
 # See LICENSE for more details.
 #
-# Copyright: 2016 IBM
+# Copyright: 2025 IBM
 # Author: Basheer Khadarsabgari <basheer@linux.vnet.ibm.com>
+# Author: R Nageswara Sastry <rnsastry@linux.ibm.com>
```

Comment:
are we adding  the Author Signature as we have a new maintainer of this Test, if not, please consider only this added if a significant change is made in code